virt/virt-4-8-release-notes.adoc

```diff
@@ -64,6 +64,12 @@ The SVVP Certification applies to:
 
 * {VirtProductName} now supports third-party Container Network Interface (CNI) plug-ins that are link:https://access.redhat.com/articles/5436171[certified by Red Hat] for use with {product-title}.
 
+[id="virt-4-8-quick-starts"]
+=== Quick starts
+
+* Quick start tours are available for several {VirtProductName} features. To view the tours, click the *?* icon in the masthead of the {VirtProductName} console and select *Quick Starts*.
```

Comment:
Does the **?** icon have a generic name to go with its function or does it use a tooltip? Per ISG, the graphic for the UI element should come after the label or tooltip name. For example, you could say, click the **Help** icon **?**.

Reply:
great, thanks.